test/integration/helpers.go

```diff
@@ -203,35 +203,52 @@ func clusterLogs(t *testing.T, profile string) {
 
 	t.Logf("-----------------------post-mortem--------------------------------")
 	t.Logf("<<< %s FAILED: start of post-mortem logs <<<", t.Name())
-	t.Logf("-------------------post-mortem minikube logs----------------------")
+	t.Logf("------++> post-mortem[%s]: minikube logs", t.Name())
+
 	rr, err := Run(t, exec.Command(Target(), "-p", profile, "logs", "--problems"))
 	if err != nil {
 		t.Logf("failed logs error: %v", err)
 		return
 	}
 	t.Logf("%s logs: %s", t.Name(), rr.Output())
 
-	t.Logf("------------------post-mortem api server status-------------------")
+	t.Logf("------++> post-mortem[%s]: disk usage", t.Name())
+	rr, err = Run(t, exec.Command(Target(), "-p", profile, "ssh", "df -h /var/lib/docker/overlay2 /var /; du -hs /var/lib/docker/overlay2"))
+	if err != nil {
+		t.Logf("failed df error: %v", err)
+	}
+	t.Logf("%s df: %s", t.Name(), rr.Stdout)
+
 	st = Status(context.Background(), t, Target(), profile, "APIServer")
 	if st != state.Running.String() {
 		t.Logf("%q apiserver is not running, skipping kubectl commands (state=%q)", profile, st)
 		return
 	}
-	t.Logf("--------------------post-mortem get pods--------------------------")
+
+	t.Logf("------++> post-mortem[%s]: get pods", t.Name())
 	rr, rerr := Run(t, exec.Command("kubectl", "--context", profile, "get", "po", "-A", "--show-labels"))
 	if rerr != nil {
 		t.Logf("%s: %v", rr.Command(), rerr)
 		return
 	}
 	t.Logf("(dbg) %s:\n%s", rr.Command(), rr.Output())
-	t.Logf("-------------------post-mortem describe node----------------------")
+
+	t.Logf("------++> post-mortem[%s]: describe node", t.Name())
 	rr, err = Run(t, exec.Command("kubectl", "--context", profile, "describe", "node"))
 	if err != nil {
 		t.Logf("%s: %v", rr.Command(), err)
 	} else {
 		t.Logf("(dbg) %s:\n%s", rr.Command(), rr.Output())
 	}
-	t.Logf("------------------------------------------------------------------")
+
+	t.Logf("------++> post-mortem[%s]: !running pods", t.Name())
+	rr, err = Run(t, exec.Command("kubectl", "--context", profile, "describe", "po", "-A", "--field-selector=status.phase!=Running"))
```

Comment:
I think the --field-selector works only on get pods, and then once u have the list, u can iterate through them.

Reply:
```
 $ kc get po -A --field-selector=status.phase=Running
NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE
kube-system   coredns-66bff467f8-cpzcv           0/1     Running   0          20s
kube-system   coredns-66bff467f8-lnb2m           1/1     Running   0          20s
kube-system   etcd-minikube                      1/1     Running   0          35s
kube-system   kindnet-wrpm7                      1/1     Running   0          20s
kube-system   kube-apiserver-minikube            1/1     Running   0          35s
kube-system   kube-controller-manager-minikube   1/1     Running   0          35s
kube-system   kube-proxy-cxfqr                   1/1     Running   0          20s
kube-system   kube-scheduler-minikube            1/1     Running   0          35s
kube-system   storage-provisioner                1/1     Running   0          32s
medmac@~/workspace/minikube (kic_stop_improve) $ kc describe po -A --field-selector=status.phase=Running
Error: unknown flag: --field-selector
See 'kubectl describe --help' for usage.
```

Reply:
Bummer. Will leave the iteration for a subsequent PR.